### ISSUE TITLE
MAJOR: Create Acccount

### ISSUE THREAD

**Issue by magalj** [DailyGrindINFO442/info442sp20#15]
Create Account
Major
Description:
When creating a new account, I tried both a fake email, real uw email, and was unable to access the site. I cleared cache and that also did not work. I tried it on edge, and it didn’t work either.
Reproduction:
1.	Click Create account
2.	Fill in form fields with uw email
3.	Click on grey create account button

**Comment by gcaelin30** [comment]
Yeah I couldn't even explore the site since it just asks me to Sign In
But with what cuz I can't create an account nor does it show me where
